### ISSUE TITLE
More feedback when waiting for server

### ISSUE THREAD

**Issue by tgetgood** [openmindanesth/openmind#65]
When saving extracts, changes, comments, etc. the client waits for the server to acknowledge something is really saved before showing it. The pause is brief enough and you can go on using the page in the meantime, but it's not clear that anything is happening because there's absolutely no feedback.

We should have at least spinners, or grey out the input boxes. There should also be a notification of some sort (banner maybe) that extracts have been successfully created, etc..

**Comment by hlesaint** [comment]
The blue glow around the "save" / "create" button is already helpful. A brief message below the button may provide additional clarity: "extract created", "extract modified".

**Comment by tgetgood** [comment]
As part of fixing the editing workflow I added a banner that indicates that the server received your changes. Once your changes are committed to the database, they automatically update in the interface, but there's no other indication. I think that's sufficient. 

I've also added a spinning cursor for the few cases where we have to wait for the server for something (like searching for an article in pubmed).

Do you think these changes suffice?

**Comment by hlesaint** [comment]
Absolutely. I think the banner could be a bit more adapted to the overall color scheme and style (maybe it could be a bit thinner? the color a bit more pastel?). But this is a luxury problem.
